libhb/decvobsub.c

```diff
@@ -603,7 +629,16 @@ static hb_buffer_t * Decode( hb_work_object_t * w )
     hb_job_t * job = pv->job;
 
     /* Get infos about the subtitle */
-    ParseControls( w );
+    if ( ParseControls( w ) )
+    {
+        /*
+         * Coudln't parse the info
+         */
+        hb_deep_log( 2, "decvobsub: Could not parse info!" );
+
+        hb_buffer_close( &pv->buf );
```

Comment:
pv->size_sub is already set to zero after Decode() is called on line 177, so it should be ok.